### PR TITLE
Edits Nomination Pools

### LINKED ISSUE #5777: Issue with the max number of nominators in the electing set
"Due to the current runtime constraints, Polkadot can only handle [object Object] nominators comfortably in the [electing set](https://wiki.polkadot.network/docs/learn-nominator#staking-election-stages)."

https://wiki.polkadot.network/docs/learn-nomination-pools

### PR DESCRIPTION
**Pull request by filippoweb3**

### CODE REVIEW

**Comment by DrW3RK** [comment]
Looks like the restrictions have been lifted https://forum.polkadot.network/t/next-revision-of-increasing-nomination-pool-limits/2498/13?u=radha

image

**Comment by laboon** [comment]
@DrW3RK That's for the number of nomination pools / nomination pool members, not the the number of nominators themselves.

**Comment by DrW3RK** [comment]
@laboon Yeah, I looked at the PR title of "Edits to Nomination Pools" and shared some context related to it.

File: docs/learn/learn-nomination-pools.md [hunk 1]
```diff
@@ -20,23 +20,13 @@ Note that rewards are not guaranteed for those pools that do not have enough bon
...
-- {{ polkadot: No limit on :polkadot }}
-  {{ kusama: There can be a maximum of <RPC network="kusama" path="query.nominationPools.maxPoolMembersPerPool" defaultValue={1024} /> :kusama }}
-  members per pool.
+There are currently
+{{ polkadot: <RPC network="polkadot" path="query.nominationPools.counterForPoolMembers" defaultValue={4376} /> :polkadot }}
+{{ kusama: <RPC network="kusama" path="query.nominationPools.counterForPoolMembers" defaultValue={389} /> :kusama }}
+members in
+{{ polkadot: <RPC network="polkadot" path="query.nominationPools.lastPoolId" defaultValue={80} /> :polkadot }}
+{{ kusama: <RPC network="kusama" path="query.nominationPools.lastPoolId" defaultValue={115} /> :kusama }}
+pools.
```
Comment (profradha) [inline docs/learn/learn-nomination-pools.md]:
```suggestion
pools. There is no limit to the number of pools or pool members per pool. 
```

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**